### ISSUE TITLE
Tensorboard debugger setup with Keras and tf-slim

### ISSUE THREAD

**Issue by realnima** [tensorflow/tensorboard#3133]
Hello,
I am having a problem with tf-slim training. The loss function stays constant throughout training, which as I understand it, maybe a Nan problem.
I am trying to setup the visual debugger. While tensorboard is working properly in realtime during training, the debugger is not.

I am following [this example](https://coral.ai/docs/edgetpu/retrain-classification/).

To setup the debugger:
```
tensorboard --logdir ./transfer_learn/train/ --debugger_port 7000
```

I receive no errors. The tensorboard works but debbuger tab does not.
I have verified the socket  bindings using the command below and they seem to be correct:
```
sudo lsof -i -P -n
```

I believe my problem is that I am not placing the code snippets at the right place:
```
tf.Session:
import tensorflow as tf
from tensorflow.python import debug as tf_debug

sess = tf.Session()
sess = tf_debug.TensorBoardDebugWrapperSession(sess, "odin-exp-nima-tf1-5e:7000")
sess.run(my_fetches)
          
Estimator | MonitoredSession:
import tensorflow as tf
from tensorflow.python import debug as tf_debug

hook = tf_debug.TensorBoardDebugHook("odin-exp-nima-tf1-5e:7000")
my_estimator.fit(x=x_data, y=y_data, steps=1000, monitors=[hook])
            
Keras Model:
import tensorflow as tf
from tensorflow.python import debug as tf_debug
import keras

keras.backend.set_session(
    tf_debug.TensorBoardDebugWrapperSession(tf.Session(), "odin-exp-nima-tf1-5e:7000"))
# Define your keras model, called "model".
model.fit(...)
```

The tutorial I am following is using keras and the model provided does not have a .fit or run in it.
I tried a few places but none worked.

My question is:
For a tf-slim and the tutorial suggested in the link, where should I place the code snippet?

Any help or suggestions are much appreciated.
Thank you.

**Comment by realnima** [comment]
I was able to resolve this issue. 
Thanks.
